### DISCUSSION TITLE
Has this project been abandoned?

### DISCUSSION THREAD

**Discussion by donnyv** [uvdesk/community-skeleton#727] [General]
I saw somewhere that know of the devs or anyone were responding anymore to requests or questions? Has the community edition been abandoned? I also noticed it hasn't been updated in almost a year.

**Comment by grayfoxcomputing** [comment]
This project looked so promising, but it looks like it's over. I was going to use it in my environment and while spinning it up noticed that the dependencies and modules are dated. Now I'm going to give REI3 a try.

I hope the development on this project is revived soon so I can give it a chance.

**Comment by papnoisanjeev** [comment]
@donnyv @grayfoxcomputing 
We have released a new version of project(v1.1.7) with many updates and bug fixes.
You can check [here](https://github.com/uvdesk/community-skeleton/releases/tag/v1.1.7).
